backend/module/eCampCore/src/EntityService/ContentNodeService.php

```diff
@@ -133,4 +143,14 @@ protected function validateEntity(BaseEntity $entity): void {
         $strategy = $this->contentTypeStrategyProvider->get($contentNode->getContentType());
         $strategy->validateContentNode($contentNode);
     }
+
+    protected function getContentNodeRepository(): ContentNodeRepository {
```

Comment:
Is lazy loading of the repository necessary?
Wouldn't it be simpler to just get it in the constructor?
(you shouldn't do anything in the constructor, but we don't have factories for these repositories.)

Reply:
Glad you asked. Since this service is used only in a very specific case (only when a new ContentNode is created, and no position is specified), I thought I'd make it lazy. But I can put it in the constructor if you think that's better, no hard feelings.